### DISCUSSION TITLE
Fixing the Levels: Excel and for all things level related

### DISCUSSION THREAD

**Discussion by shoc71** [shoc71/dont-stop-now#220] [General]
https://docs.google.com/spreadsheets/d/1k4hGtyzFeRAulf-8Fta5CI69PNqdWIoKs4LxAmOilHs/edit?usp=sharing

(There are two sheets)

**Comment by slysnivy** [comment]
@shoc71 I added more ideas to the excel. We should also update it to be up to date.

**Comment by shoc71** [comment]
most, if not, all the level ideas you mentioned could be turned into their own series of levels with their own story to tell. I'll try and make something out of it.

**Comment by shoc71** [comment]
While I agree about keeping it up to date, I have been making a physical list of notes about what levels editing, touching, and all that other stuff but I will update that excel for things to look better and more smoothly.
